control_fun.h

```diff
@@ -228,10 +228,12 @@ class PI2Controller {
     float step(float desired, float measured);
```

Comment:
A comment on this PI2Controller class explaining it's used for current control with an output of voltage would help give the context behind the `is_voltage_saturated` function